pkg/kubelet/container_bridge.go

```diff
@@ -82,6 +82,16 @@ func ensureCbr0(wantCIDR *net.IPNet) error {
 		}
 		return createCBR0(wantCIDR)
 	}
+	// Put the container bridge into promiscuous mode to force it to accept hairpin packets.
```

Comment:
This seems incorrect ??
You want the logic in the createCBR0 function. 
Else this code will only be called the second time around when a cbr0 already exists.

Is that what you wanted ?